src/customHooks/useAuthorize.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
+
+const URL = 'http://localhost:5020/auth';
+const token = 'token';
```

Comment:
1. the convention for constants is UPPERCASE, so TOKEN
2. is this the best place to store a global variable for your app?
3. you can simply to `export const TOKEN `here

Reply:
2. Maybe in a separate file just for global variables..?

Reply:
sounds like a plan